src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/adjust_brgemm_copy_b_loop_ports.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "snippets/lowered/pass/pass.hpp"
+#include "snippets/lowered/loop_info.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace pass {
+
+/**
+ * @interface AdjustBrgemmCopyBLoopPorts
+ * @brief BrgemmCopyB is located outside of blocking loops and repacks input data into a blocked layout.
+ *        This layout should be accounted for when we increment BrgemmCopyB data pointers. This pass
+ *        Finds loop ports connected to BrgemmCopyB and sets appropriate pointer increments.
```

Comment:
Yes, but not entirely.
A proper handling of blocked layouts would automatically enable N loop adjustments (so called VNNI format).
However, the present implementation also modifies K-blocking loops, because of the current implementation of BrgemmCopyB emitter & executor on our side (padding to the full 2x32 or 4x64 block). I believe it is not strictly necessary anymore, but an additional investigation is required (remember recent discussion in the snippets chat?)